src/core/main.c

```diff
@@ -1364,7 +1363,6 @@ int main(int argc, char *argv[]) {
 
                 /* Running outside of a container as PID 1 */
                 arg_system = true;
-                make_null_stdio();
```

Comment:
yes, of course, a large chunk of programs write interesting stuff to stderr or stdout, not syslog. UI programs do that anyway, but also a lot of stuff that runs in the background, for example cron jobs and suchlike. Moreover, even system services that normally log to syslog directly print errors in their command line and suchlike to stderr, and we want to cover that. So yupp, we should always collect stdout/stderr output by default, unless there's a really good reason not to do this in specific cases.